src/osm_configurator/view/utilityframes/tag_list_priority_frame.py

```diff
@@ -256,64 +260,77 @@ def _entry_button_pressed(self, button_id: int):
                              fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_ACTIVE.value)
 
         # Disabling the pressed button
-        self._entry_button_list[button_id].configure(state="disabled",
-                                                     text_color=button_constants_i.ButtonConstants.BUTTON_TEXT_COLOR_DISABLED.value,
-                                                     fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_DISABLED.value)
+        self._entry_button_list[button_id].configure(
+            state="disabled",
+            text_color=button_constants_i.ButtonConstants.BUTTON_TEXT_COLOR_DISABLED.value,
+            fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_DISABLED.value
+        )
+
         self._selected_button_id: int = button_id
         self._last_pressed_entry_button: customtkinter.CTkButton = self._entry_button_list[button_id]
 
         # telling parent what entry was pressed
-        # since buttons and entries alling in theri id, we can just use the button_id to find it
+        # since buttons and entries in the id, we can just use the button_id to find it
```

Comment:
sentence now doesn't make any sense, you made it worse here, would use old sentence and fix the typo instead